middleware/cron.job.js

```diff
@@ -0,0 +1,43 @@
+const cron = require("node-cron");
```

Comment:
you dont work with the updated version of develop,cronjob.js is no longer at middleware folder, its in service